src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -114,17 +115,17 @@ We'll use the `<script>` tag to link to the Infusion library:
 <script type="text/javascript" src="lib/infusion/infusion-custom.js"></script>
```

Comment:
I think this code snippet should say `<script type="text/javascript" src="lib/infusion/infusion-all.js"></script>` since that is the most likely scenario if the user is following the documentation.